Nodejs/Product/TestAdapter/TestReceiver.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter;
+
+namespace Microsoft.NodejsTools.TestAdapter
+{
+    internal sealed class TestReceiver : ITestCaseDiscoverySink
+    {
+        public readonly List<TestCase> Tests = new List<TestCase>();
```

Comment:
Arguably, this should be exposed as an `IReadOnlyList`, but it's internal so I don't care very much.

Reply:
Made class private